.github/workflows/build.yml

```diff
@@ -20,8 +20,9 @@ jobs:
           - 10-stretch-slim
           - 12-stretch-slim
           - 14-stretch-slim
+          - 16-stretch-slim
         include:
-          - tag: 14-stretch-slim
+          - tag: 16-stretch-slim
```

Comment:
When we pull the _latest_ tag or don't define a tag, this will now be using Node 16. Is that expected?

Reply:
Yes, is that okay?  Do you see any reason to stick with `14` as the latest?

Reply:
Wanted to make sure you understood this change. I think it's fine to bump as the latest

Reply:
On second thought, I'm reverting this back to `14`.  `16` should be tested in the wild before committing it to `latest`.  I'm sure `images` is the only service using this image and we specify the tag to 12... but best to be safe.